lib/views/push-pull-view.js

```diff
@@ -2,37 +2,196 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
+import {RemotePropType, BranchPropType} from '../prop-types';
+
+
```

Comment:
🚨 🎨 Looks like this file has a few extra newlines in it. (I'm surprised the linter didn't complain, actually.)